src/main/scala/nl/knaw/dans/bag/DepositProperties.scala

```diff
@@ -39,6 +39,12 @@ case class DepositProperties(creation: Creation = Creation(),
                              springfield: Springfield = Springfield(),
                              staged: Staged = Staged()) {
 
+  //TODO shouldn't this always be 'deposit.properties'? in the current Deposit.baseDir?
```

Comment:
or am i missing something?

Reply:
I think this is an old comment I forgot to remove. The current philosophy is that `DepositProperties` doesn't know it belongs to a bag, or at which location it is stored. That information is only known by the `Deposit` that holds this instance of `DepositProperties`. So it is correct that this `save` method takes the `file: File` as an argument.